src/test/java/core/basesyntax/WorkWithFileTest.java

```diff
@@ -28,7 +29,11 @@ public void clearResults() {
 
     @Test
     public void getStatisticAboutApple() {
-        workWithFile.getStatistic("apple.csv", APPLE_RESULT_FILE);
+        try {
+            workWithFile.getStatistic("apple.csv", APPLE_RESULT_FILE);
+        } catch (FileNotFoundException e) {
```

Comment:
Catching `FileNotFoundException` is not appropriate here because the `getStatistic` method can throw `IOException`. Consider catching `IOException` instead.